### DISCUSSION TITLE
Tidsbegrænsning

### DISCUSSION THREAD

**Discussion by mtjoernelund** [scaarup/aula#145] [General]
Er lidt usikker på om jeg gør noget galt her, eller om det er en begrænsning i den data der kommer ind. 

Jeg forsøger at lave et service call:

> service: calendar.list_events
> target:
>   entity_id: calendar.skoleskema
> data:
>   duration:
>     hours:24

(sorry, indrykning er korrekt men kan ikke formatere yaml her…)

Og det virker for såvidt fint, med den undtagelse at “duration” ikke bliver accepteret. Jeg får events så langt frem som de er tilgængelige. Jeg har også lagt mærke til, at når jeg forsøger at begrænse antal dage der bliver vist når jeg anvender “Atomic Calendar Revive” bliver denne begrænsning heller ikke respekteret.

Data ser ellers fint ud og hvis jeg ser dem i HA’s kalender står det hele rigtigt. Er der et eller andet jeg misser?

**Comment by scaarup** [comment]
I dit eksempel mangler du at sætte 'start_date_time'

**Reply by scaarup** [reply]
Men det samme sker for mig her. Jeg ved ikke hvad der foregår.

**Reply by mtjoernelund** [reply]
‘start_date_time_’ skulle ikke være nødvendigt. Hvis ikke den er sat er default ‘now()’. Jeg prøvede lige at teste med en anden kalender, der virker præcis samme service kald som ventet, altså den begrænser til begivenheder indenfor det angivne antal timer. 

Men det er også interessant at jeg ikke kan se nogen forskel i output. Det er altså ikke fordi dator/tidsformat i start eller end er forkert. Det er helt identisk ift hvad jeg får fra kalendre der faktisk respekterer den angivne tidsgrænse. Så der må foregå et eller andet i baggrunden. Jeg ved bare ikke rigtigt hvor jeg skal lede…

**Comment by scaarup** [comment]
Men nu poppede denne her op i HA:
![image](https://github.com/scaarup/aula/assets/8055470/6886aea0-0073-4c3e-9578-b05185680b40)
